albumentations/augmentations/transforms.py

```diff
@@ -605,7 +606,104 @@ def get_transform_init_args_names(self):
         return ("mean", "std", "max_pixel_value")
 
 
-class Cutout(ImageOnlyTransform):
+class RandomErasing(DualTransform):
+    """ Randomly selects a rectangle region in an image and erases its pixels.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+    Args:
+         probability: The probability that the Random Erasing operation will be performed.
+         sl: Minimum proportion of erased area against input image.
+         sh: Maximum proportion of erased area against input image.
```

Comment:
What the difference between `sl` and `sh`?

Reply:
The erased area will be in [sl * area, sh * area] per hole.

Reply:
Just add this info into docstring.